apps/meteor/server/models/raw/PersistentQueue.ts

```diff
@@ -0,0 +1,19 @@
+import type { Db, IndexDescription } from 'mongodb';
+import type { IPersistentQueueModel } from '@rocket.chat/model-typings';
+
+import { BaseRaw } from './BaseRaw';
+
+export class PersistentQueueModel extends BaseRaw<any> implements IPersistentQueueModel {
```

Comment:
I think we should provide the correct typings for what's being handled here, instead of `any`. As the example here: https://github.com/RocketChat/Rocket.Chat/blob/develop/apps/meteor/server/models/raw/MatrixBridgedRoom.ts#L7
and the type: https://github.com/RocketChat/Rocket.Chat/blob/develop/apps/meteor/server/models/raw/MatrixBridgedRoom.ts#L1